src/index.html

```diff
@@ -4,10 +4,29 @@
 <html lang="en">
 	<head>
 		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" href="./style.css" />
 		<title>Webpack todo</title>
 	</head>
 	<body>
-		<h1>Hello webpack!</h1>
+		<div class="container">
+			<div class="todo-title-container">
+				<h1 class="todo-title">Today's To do</h1>
+				<img
+					class="refreshBtn"
+					src="https://img.icons8.com/ios-glyphs/30/null/refresh--v1.png" />
+			</div>
+			<div class="todo-input">
+				<input type="text" id="textInput" placeholder="add list here" />
+
+				<img
+					id="addBtn"
```

Comment:
- [optional] please increase the size of the add icon to make it more visible. You can use a Plus sign(icon) there instead, since it's more user-friendly as an addition button